### PR TITLE
Use eslint-config-nextcloud

### LINKED ISSUE #837: Use nextcloud eslint config
See https://github.com/nextcloud/eslint-plugin

### PR DESCRIPTION
**Pull request by raimund-schluessler**
With this PR the Tasks code now follows the `eslint-config-nextcloud` rules. Closes #837 

@skjnldsv This one is for you 😉

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,11 +1,4 @@
 module.exports = {
-	root: true,
-	env: {
-		browser: true,
-		es6: true,
-		node: true,
-		jest: true
-	},
 	globals: {
 		t: false,
```
Comment (skjnldsv) [inline .eslintrc.js]: is this needed?

File: src/components/Confirmation.vue [hunk 2]
```diff
@@ -48,21 +48,20 @@ License along with this library.  If not, see <http://www.gnu.org/licenses/>.
 import ClickOutside from 'vue-click-outside'
 
 export default {
-	name: 'PopoverMenu',
```
Comment (skjnldsv) [inline src/components/Confirmation.vue]: No name?

File: .eslintrc.js (hunk 1 above)
Comment (raimund-schluessler) [reply .eslintrc.js]: Not sure, I will check.

File: src/components/TaskBody.vue [hunk 3]
```diff
@@ -21,18 +21,16 @@ License along with this library.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
 <template>
-	<li	v-show="showTask"
```
Comment (skjnldsv) [inline src/components/TaskBody.vue]: I would never have caught that xD

File: src/components/Confirmation.vue (hunk 2 above)
Comment (raimund-schluessler) [reply src/components/Confirmation.vue]: I can add it back, but I think it is not necessary. The directive works and doesn't raise an error in the console. Also, this directive won't be used anymore after #818 is merged.

File: src/components/TaskBody.vue (hunk 3 above)
Comment (raimund-schluessler) [reply src/components/TaskBody.vue]: Me neither 😄

**Comment by skjnldsv** [comment]
> Btw, with the new github actions, you can get linting comments in your pr directly 
> I can provide a pr

https://github.com/nextcloud/tasks/pull/844

**Pull request merged**